backend/src/bundles/achievements/achievement.controller.ts

```diff
@@ -213,6 +226,40 @@ class AchievementController extends BaseController {
         const { params } = options;
         return this.findByUserId(params.id);
     }
+
+    /**
+     * @swagger
+     * /api/v1/achievements/current-user:
```

Comment:
we have an endpoint for finding everything by user id
`/api/v1/achievements/user/{id}`